src/sass/base/_utility.scss

```diff
@@ -1,9 +1,36 @@
 .u-heading-l {
-    font-size: 4rem;
+    font-size: 3.6rem;
+    font-family: $font-secondary;
+    font-weight: 400;
+    color: $white;
+
+    &--black {
+        color: $black-heading;
+    }
 }
 .u-heading-md {
-    font-size: 3rem;
+    font-size: 2rem;
+    font-weight: 700;
+    color: $gray;
+
+    &--black-heading {
+        color: $black-heading;
+    }
+
+    &--black {
+        color: $black;
+    }

```

Comment:
We have --black-heading and --black. What about --dark-gray for example if it is not black?